server/src/main/java/org/elasticsearch/search/aggregations/LeafBucketCollector.java

```diff
@@ -36,36 +42,6 @@ public boolean isNoop() {
         }
     };
 
-    public static LeafBucketCollector wrap(Iterable<LeafBucketCollector> collectors) {
```

Comment:
Was this just not being used?  Seems a little surprising to be removing it in a docs change.

Reply:
Sorry. Yeah, it was not being used at all. I can slice out the change though.